packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -297,6 +297,21 @@ void main() {
     ...noColorTerminalOverride,
   });
 
+  testUsingContext('create --offline success', () async {
```

Comment:
Sorry, I should have caught it before, but I believe that this integration test is not needed because we already have: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/commands.shard/permeable/create_test.dart#L1871